src/Multiplexer/index.js

```diff
@@ -85,17 +79,41 @@ const views = mapObjectToObject(
     ),
 );
 
+const ChromeDownloadLink = () => (
+    <a href="https://www.google.com/chrome/">
+        {_ts('multiplexer', 'hereLabel')}
+    </a>
+);
+
 // NOTE: withRouter is required here so that link change are updated
 @withRouter
 @connect(mapStateToProps, mapDispatchToProps)
 export default class Multiplexer extends React.PureComponent {
     static propTypes = propTypes;
 
+    constructor(props) {
+        super(props);
+        const browser = browserDetect();
```

Comment:
Looks like this could have been a separate component.

Reply:
Which one?

Reply:
I mean this whole feature of identifying this and that.
Something like <DetectChrome /> which goes on render